cmd/flipt/default.go

```diff
@@ -0,0 +1,6 @@
+//go:build !linux
+// +build !linux
+
+package main
+
+var defaultCfgPath string
```

Comment:
i guess another alternative is to check the OS at runtime instead of using build tags and an empty var here, anyone have a pref over one or the other?

Reply:
I think this is cool. It follows the pattern of the other things we are doing with build tags as well, so just for consistency.

Reply:
wdyt @GeorgeMac ?

Reply:
yeah was debating on if this should be at `Info` level instead of `Warning`